packages/lumos/data_stream/activity_logs/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,74 @@
+---
+description: Pipeline for processing Lumos Activity Logs
+processors:
+  - set:
+      field: ecs.version
+      value: "8.11.0"
+  - json:
+      field: message
+      target_field: json
```

```suggestion
      target_field: lumos.activity_logs
```
This way you don't need to do all the individual renames.